benches/bench.rs

```diff
@@ -77,7 +77,7 @@ const NUM_COMPONENTS: usize = 200;
 
 // --------------
 
-#[derive(SystemData)]
```

Comment:
It's kinda unfortunate that the common case becomes longer after this, but I don't know how to avoid that.

Reply:
Yeah, just mentioned that on Gitter, I don't like it either. I think I'm gonna rename `StaticSystemData` back to `SystemData` and call the other one `DynamicSystemData`